distributed/comm/ucx.py

```diff
@@ -325,12 +325,13 @@ async def read(self, deserializers=("cuda", "dask", "pickle", "error")):
             await self.ep.recv(header)
             header = struct.unpack(header_fmt, header)
             cuda_frames, sizes = header[:nframes], header[nframes:]
-        except (
-            ucp.exceptions.UCXCloseError,
-            ucp.exceptions.UCXCanceled,
-        ) + (getattr(ucp.exceptions, "UCXConnectionReset", ()),):
+        except BaseException as e:
+            # In addition to UCX exceptions, may be CancelledError or a another
+            # "low-level" exception. The only safe thing to do is to abort.
+            # (See also https://github.com/dask/distributed/pull/6574).
             self.abort()
-            raise CommClosedError("Connection closed by writer")
+            raise CommClosedError("Connection closed by writer.\n"
+                                  "Inner exception: {e!r}")
```

Comment:
Sorry, my suggestion missed the f-string prefix:
```suggestion
                                  f"Inner exception: {e!r}")
```